user_stories.md

```diff
@@ -0,0 +1,18 @@
+### Bus Mall
+### User Stories
+
```

Comment:
Add some more user stores. You need 4-5 for users, developers and market researcher. Split into sections with a header for each.